### ISSUE TITLE
question: how to express that ordering doesn't matter?

### ISSUE THREAD

**Issue by tommyjcarpenter** [taverntesting/tavern#504]
I have a test:

```
  - name: instance list
    request:
      url: http://localhost:10000/a1-p/policytypes/1006001/policies
      method: GET
    response:
      status_code: 200
      body: [qt1, qt2]
```

how do I express that `[qt1, qt2]` OR `[qt2,qt1]` is OK? Is there an "unordered_ok" or some equivalent flag?

**Comment by michaelboulton** [comment]
There isn't something to do that at the moment but it might be possible with the changes to the strictness settings in the 1.0 release

**Comment by MistaTwista** [comment]
@michaelboulton hey 👋 is there any changes in this question?

**Comment by michaelboulton** [comment]
https://github.com/taverntesting/tavern/pull/604 I added a branch that should let you do this by specifying `json:list_any_order` which is basically an even less strict version of `strict:off` and will ignore the order of any list items. If you want to give it a try it's there, I need to go back and add a lot more tests to it.

**Comment by ryaoik** [comment]
@michaelboulton Hi! We are currently facing a similar problem.The #604 pull request seems to help solve that.
It is now 2021, when will #604 be merged and released and will this issue be resolved?

**Comment by michaelboulton** [comment]
I've merged that one and uploaded it to pypi as 1.16.

For a bit of history - this was actually an issue in 2016 when tavern was first being written for an internal use case and it took 5 years to fix!
